Header header

float64 actual_x_av
float64 actual_y_av
float32 actual_yaw_av
float64 actual_speed_av
float64 actual_steer_av
float64 actual_action_av
float32[] path_x_av
float32[] path_y_av

float64 target_x_av
float64 target_y_av
float32 target_yaw_av
float64 target_speed_av
float64 target_steer_av

float64 actual_x_other
float64 actual_y_other
float32 actual_yaw_other
float64 actual_speed_other

float32 game_reward
float32 game_obstacle_x
float32 game_obstacle_y
float32 game_case_other
float32 game_final_x
float32 game_final_y